src/parser/parsers/heroes.py

```diff
@@ -84,7 +84,52 @@ def run(self):
 
                 all_hero_stats[hero_key] = json_utils.sort_dict(hero_stats)
 
-        return all_hero_stats
+        # Write non-constant stats to json file
+        non_constants = self._write_non_constants_stats(all_hero_stats)
+
+        return all_hero_stats, non_constants
+
+    def _write_non_constants_stats(self, all_hero_stats):
+        """
+        Writes list of non-constant stats to json file.
+
+        Returns non_constants_stats dict, where elements are true if they are non-constant.
+
+        Non-constant stats are ones that will be displayed
+        on the deadlocked.wiki/Hero_Comparison page, among others in the future.
+        """
```

Comment:
I think be clearer as to what non-constant means. Eg. `a "non-constant" stat is one that differs between heroes and is therefore suitable to be displayed on the Hero_Comparison page`

Reply:
Renamed from "non-constant-stats" to "meaningful-stats"

Reply:
Still requires an explanation as to what meaningful or non-constant means

Reply:
added